src/System.Console/tests/System.Console.Tests.builds

```diff
@@ -1,13 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
-  <ItemGroup>
-    <Project Include="System.Console.Tests.csproj" />
```

Comment:
Before we start deleting builds files we need to be sure we capture the information in a BuildConfigurations property. @chcosta has generated some of them in his PR but I don't know if he has them all yet.